File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Orca/ORCATransferOrdersErrMsgInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Orca
{
    [Serializable]
    public class ORCATransferOrdersErrMsgInfo
    {
        private String _hangmogCode;
        private String _hangmogName;
        private String _errCode;

        public String HangmogCode
        {
            get { return this._hangmogCode; }
            set { this._hangmogCode = value; }
        }

        public String HangmogName
        {
            get { return this._hangmogName; }
            set { this._hangmogName = value; }
        }

        public String ErrCode
        {
            get { return this._errCode; }
            set { this._errCode = value; }
        }

        public ORCATransferOrdersErrMsgInfo() { }

        public ORCATransferOrdersErrMsgInfo(String hangmogCode, String hangmogName, String errCode)
        {
            this._hangmogCode = hangmogCode;
            this._hangmogName = hangmogName;
            this._errCode = errCode;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/orca/ORCATransferOrdersErrMsgInfo.java
package nta.med.data.model.ihis.orca;

public class ORCATransferOrdersErrMsgInfo {
	private String hangmogCode ;
	private String hangmogName ;
	private String errCode      ;
	public ORCATransferOrdersErrMsgInfo(String hangmogCode, String hangmogName,
			String errCode) {
		super();
		this.hangmogCode = hangmogCode;
		this.hangmogName = hangmogName;
		this.errCode = errCode;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getHangmogName() {
		return hangmogName;
	}
	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}
	public String getErrCode() {
		return errCode;
	}
	public void setErrCode(String errCode) {
		this.errCode = errCode;
	} 

}
